src/store/redis/redis_nodeset.h

```diff
@@ -129,6 +132,36 @@ typedef struct {
 } redis_nodeset_dbg_range_tree_t;
 #endif
 
+typedef enum {
+   //must start at 0
```

Comment:
It says it must start at zero but the enum starts at `-1`

Reply:
Indeed, I'll document this better -- the actual _command codes_ must start at 0, `NCHAN_REDIS_CMD_ANY` is a catch-all group and not an individual command.

Reply:
done in 9c8d8d3da0a534f